### PR TITLE
Stable

### PR DESCRIPTION
**Pull request by sarika296**
Testing Redux Pull Requst.

### CODE REVIEW

**Review by Ajayff4 — APPROVED** [review]
Ok you can go ahead.

**Comment by sarika296** [comment]
> Ok you can go ahead.

ok

File: src/components/CakeContainer.js [hunk 1]
```diff
@@ -1,26 +1,28 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { connect } from 'react-redux'
 import { buyCake } from "../redux"
 
 function CakeContainer(props) {
+    const [number, setNumber] = useState()
```
Comment (Ajayff4) [inline src/components/CakeContainer.js:6]: Initial value should be in the useState() as parameter

File: src/components/ChocolateContainer.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react'
+import { connect } from 'react-redux'
+import { buyChocolate } from "../redux"
+
+function ChocolateContainer(props) {
+    const [number, setNumber] = useState()
```
Comment (Ajayff4) [inline src/components/ChocolateContainer.js:6]: Same mistake again, it should be useState(1);

File: src/components/iceCreamContainer.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react'
+import { connect } from 'react-redux'
+import { buyIceCream } from "../redux"
+
+function IceCreamContainer(props) {
+    const [number, setNumber] = useState()
```
Comment (Ajayff4) [inline src/components/iceCreamContainer.js:6]: Again same mistake, you need to initialize the value to avoid undefined and null values.

**Review by Ajayff4 — COMMENTED** [review]
Fix useState()  to useState(0) for every file which uses it.

**Review by Ajayff4 — COMMENTED** [review]
Fix proposed changes.

**Pull request merged**